trunk/SUAVE/Components/Wings/All_Moving_Surface.py

```diff
@@ -0,0 +1,97 @@
+## @ingroup Components-Wings
+# All_Moving_Surface.py
+#
+# Created:  Jul 2021, A. Blaufox
+# Modified: 
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+from SUAVE.Core import Data
+
+import numpy as np
+
+# ----------------------------------------------------------------------
+#  Attribute
+# ----------------------------------------------------------------------
+
+## @ingroup Components-Wings
+class All_Moving_Surface(Data):
```

Comment:
This should inherit from Lofted Body

Reply:
I changed it. Just want to make sure that this wouldn't cause any problems with erasing/resetting variables from multiple inheritance?

Reply:
The more similar the items the easier the inheritance. But you of course can check to be sure it does as you'd expect

Reply:
Yeah it didn't cause any problems with the regression or after fiddling with it a bit at least